iaac/terraform/aws-infra/s3/variables.tf

```diff
@@ -0,0 +1,20 @@
+variable "aws_access_key" {
+  type        = string
+  description = "AWS access key to authenticate minio client"
```

Comment:
use the terms access key ID and secret access key everywhere

Reply:
done